src/clr-angular/data/datagrid/datagrid-column.spec.ts

```diff
@@ -317,6 +319,13 @@ export default function(): void {
         expect(context.clarityDirective.sortOrder).toBe(ClrDatagridSortOrder.DESC);
       });
 
+      it('add aria-label to button', function() {
+        context.testComponent.comparator = new TestComparator();
+        context.detectChanges();
+        const title = context.clarityElement.querySelector('.datagrid-column-title');
+        console.log(title, title.attributes, context.testComponent, title.attributes['aria-label'].value);
```

Comment:
Remove the log, and you need an `expect()` for this to be a real test.

Reply:
I had stashed this exact change and forgot to pop it out ... that was a lame mistake from my side.. sorry for that.